source/node_test.go

```diff
@@ -361,13 +373,20 @@ func testNodeSourceEndpoints(t *testing.T) {
 			_, err := kubernetes.CoreV1().Nodes().Create(context.Background(), node, metav1.CreateOptions{})
 			require.NoError(t, err)
 
+			if tc.exposeInternalIPv6 == nil {
+				t.Logf("WARNING: The default behavior of exposing internal IPv6 addresses will change in the next minor version. Use --expose-internal-ipv6=false flag to opt-in to the new behavior.")
```

Comment:
Found the log, but unknown configuration for me `t.Logf`??.

 It's fine to be there. Not able to find a test that capture warning message

Reply:
https://github.com/kubernetes-sigs/external-dns/blob/ecd57c86f526aebf1ecf54d0291de61ec3d48d6d/internal/testutils/log.go#L36

You could find example usages in the code

Reply:
Should I disable t.Parallel for this? Adding logs to buffer leads to race conditions on the test

Reply:
Could you try to create a completely new test just to test logging?

This parallel test run is a problem agree. Something has changed in Go 1.24, or log test helper needs adjusting

Reply:
There are examples in code aka what need to be done for parallel tests. It may work it may not

Reply:
yeah I know right.. 1.24 is seeming to do weird stuff on my mac too.. yeah will create a new set of tests just for this and see if that helps

Reply:
@ivankatliarchuk added a basic test.. not sure if this is what you had in mind.. let me know
update description and added todo tto update the external dns version too